deploy/serviceaccount.yaml

```
@@ -0,0 +1,77 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cad-sa
+  namespace: configuration-anomaly-detection
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cad-eventlistener-role
+  namespace: configuration-anomaly-detection
+rules:
+  - apiGroups:
+      - "*"
+    resources:
+      - configmaps
+      - secrets
+      - serviceaccounts
+    verbs:
+      - list
+      - watch
+      - get
+  - apiGroups:
+      - triggers.tekton.dev
+    resources:
+      - "*"
+    verbs:
+      - "*"
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - "*"
+    verbs:
+      - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: cad-eventlistener-clusterrole
+rules:
+  - apiGroups:
+      - triggers.tekton.dev
+    resources:
+      - clusterinterceptors
+    verbs:
+      - list
```

```suggestion
      - list
      - watch
```
failed by getting the error:
```
E0221 13:10:41.885069       1 reflector.go:138] github.com/tektoncd/triggers/pkg/client/informers/externalversions/factory.go:117: Failed to watch *v1alpha1.ClusterInterceptor: unknown (get clusterinterceptors.triggers.tekton.dev)
```